scripts/get-env-root.sh

```diff
@@ -30,6 +30,21 @@ apt-get install -y iverilog gtkwave
 # Nothing needed
```

Comment:
This whole script is designed to be run as root.

Maybe it should check that it is running as root and suggest using sudo otherwise?